SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Ingestion/Models/StartServiceLog.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+
```

Comment:
Yes but that's part of it- the class should also have the [JsonObject(JsonIdentifier)] attribute